content/blog/getting-started-with-hpe-greenlake-api-for-virtualization.md

```diff
@@ -0,0 +1,411 @@
+
+```json
+{
+    "associatedResources": [],
+    "childTasks": [
+        {
+            "name": "Create datastore: VVOL-ds1",
+            "resourceUri": "/data-services/v1beta1/async-operations/ad25f64d-7bd1-439e-ac31-17728e6cdbc6",
+            "type": "task"
+        }
+    ],
+    "createdAt": "2024-03-29T00:56:16.981296528Z",
+    "customerId": "eb988b5e2dcb11ec840712b3b5263ef4",
+    "displayName": "Provisioning datastore VVOL-ds1",
+    "endedAt": "2024-03-29T00:59:22.975723884Z",
+    "error": null,
+    "estimatedRunningDurationMinutes": 0,
+    "generation": 8,
+    "groups": [
+        {
+            "id": "eb988b5e2dcb11ec840712b3b5263ef4",
+            "name": "Default Group"
+        }
+    ],
+    "healthStatus": "OK",
+    "id": "9825d5e8-5213-433b-8fbf-1945bb4496e1",
+    "logMessages": [
+        {
+            "message": "Task created",
+            "timestampAt": "2024-03-29T00:56:16.98131301Z"
+        },
+        {
+            "message": "Task is running",
+            "timestampAt": "2024-03-29T00:56:16.981315803Z"
+        },
+        {
+            "message": "Preparing parameters",
+            "timestampAt": "2024-03-29T00:56:18.622573215Z"
+        },
+        {
+            "message": "Creating datastore",
+            "timestampAt": "2024-03-29T00:56:19.797636611Z"
+        },
+        {
+            "message": "Validating datastore",
+            "timestampAt": "2024-03-29T00:56:36.120087916Z"
+        },
+        {
+            "message": "Applying protection policy",
+            "timestampAt": "2024-03-29T00:57:36.332756275Z"
+        },
+        {
+            "message": "Task succeeded",
+            "timestampAt": "2024-03-29T00:59:22.975739714Z"
+        }
+    ],
+    "name": "Provisioning datastore VVOL-ds1",
+    "parentTask": null,
+    "progressPercent": 100,
+    "recommendations": [],
+    "resourceUri": "/data-services/v1beta1/async-operations/9825d5e8-5213-433b-8fbf-1945bb4496e1",
+    "rootTask": {
+        "id": "9825d5e8-5213-433b-8fbf-1945bb4496e1",
+        "name": "",
+        "resourceUri": "/data-services/v1beta1/async-operations/9825d5e8-5213-433b-8fbf-1945bb4496e1",
+        "type": "task"
+    },
+    "services": [
+        "private-cloud-business-edition"
+    ],
+    "sourceResource": {
+        "name": "VVOL-ds2",
+        "resourceUri": "/api/v1/datastores/fdcc724f-3f22-5fa6-8477-81f0fd701512",
+        "type": "Datastore"
+    },
+    "startedAt": "2024-03-29T00:56:16.981298562Z",
+    "state": "SUCCEEDED",
+    "subtreeTaskCount": 2,
+    "suggestedPollingIntervalSeconds": 30,
+    "type": "task",
+    "updatedAt": "2024-03-29T00:59:23.013384523Z",
+    "userId": "ronald.dharma@hpe.com"
+}
+```
+
+*T﻿he above figure shows the termination of machine-instance in AWS reached 100% progress and completed successfully.*
```

Comment:
Is it really the creation of a machine-instance in AWS? 
I think you meant creation of a datastore. Don;t you?